plugp100/responses/device_state.py

```diff
@@ -81,28 +81,75 @@ def try_from_json(kwargs: dict[str, Any]) -> Either['LedStripDeviceState', Excep
 @dataclass
 class DeviceInfo:
     device_id: str
+    hardware_id: str
+    oem_id: str
+    timezone: str
     firmware_version: str
     hardware_version: str
+    ip: str
     mac: str
     nickname: str
     model: str
     type: str
     overheated: bool
+
+    # wifi
+    ssid: str
     signal_level: int
     rssi: int
+
+    # location data
+    latitude: int
+    longitude: int
+    timezone: str
+    time_difference: int
+    language: str
+
+    # device state
+    power_protection_status: str
+    device_on: bool
```

Comment:
I consider `device_on` a property of a specific tapo product, for example the tapo HUB doesn't have device_on. So I think it is better to remove it from here. While other parameters you suggest are welcome, thanks

Reply:
Ok, I can change that. I suppose the `power_protection_status`, `on_time` and `auto_off*` fields are also specific to the plugs?  Just to be sure, these should go to the `PlugDeviceState` class too, right?

Reply:
I think yes

Reply:
Done!